<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boxConnection</class>
 <widget class="QGroupBox" name="boxConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="title">
   <string>Session</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="labelHostName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHostName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPortNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPortNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>boxConnection</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
